web/source/kmwbase.ts

```diff
@@ -710,7 +710,11 @@ namespace com.keyman {
 
       PKbd = PKbd || this.core.activeKeyboard;
 
-      return com.keyman.osk.VisualKeyboard.buildDocumentationKeyboard(PKbd, argFormFactor, argLayerId, this.osk.computedHeight);
+      // help.keyman.com will set this function in place, so we'll give it priority.
+      const getOskHeight = this['getOskHeight'];
+      let targetHeight = (typeof getOskHeight == 'function') ? getOskHeight() : this.osk.computedHeight || 200;
```

Comment:
```suggestion
      let targetHeight = (typeof getOskHeight == 'function' ? getOskHeight() : this.osk.computedHeight) || 200;
```

... operator precedence ftw ...